backend/admin_update_task_router.py

```
@@ -123,7 +123,7 @@ async def update_workflow_task(
     except KeyError:
         raise HTTPException(status_code=404, detail=f"Run {run_id!r} not found")
     except ValueError as exc:
-        raise HTTPException(status_code=409, detail=str(exc))
+        raise HTTPException(status_code=409, detail="Internal server error")
```

Ruff (0.15.20)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)